ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -674,6 +674,13 @@ public class KsqlConfig extends AbstractConfig {
       = "The default timeout for ASSERT SCHEMA statements if no timeout is specified "
       + "in the statement.";
 
+  public static final String KSQL_WEBSOCKET_TIMEOUT_ENABLE
+      = "ksql.websocket.timeout.enable";
+  public static final boolean KSQL_WEBSOCKET_TIMEOUT_ENABLE_DEFAULT = true;
```

Comment:
What about using what KIP-368 does? Kafka has a `connections.max.reauth.ms` that if set to `0`, then the re-auth is disabled, but if set to a positive number, then timeout will use the token expiry time or this config timeout, whichever is the lowest.

We can name this config `ksql.websocket.connection.max.reauth.ms` or `...timeout.ms`. But as we're doing this on authentication only, then `...reauth.ms` makes sense 'cause the `timeout.ms` won't work on unauthenticated connections.

Reply:
We're not supporting reauth. We're just terminating connections, so I set the config to `ksql.websocket.connection.max.timeout.ms`. The behavior is, if the connection has a token with an expiration, then close the websocket after `min(token expiration, config expiration)`. Otherwise, don't close the connection.